controller/api/models.py

```diff
@@ -526,6 +567,21 @@ def logs(self, log_lines=str(settings.LOG_LINES)):
         data = subprocess.check_output(['tail', '-n', log_lines, path])
         return data
 
+    def logs(self, log_lines=str(settings.LOG_LINES)):
+        if self._get_logging_mode() == 'standard':
```

Comment:
Is there a reason why we would want to get logs from the file system over fetching them from the logger component?

Why wouldn't we just always fetch from the logger?